stubs/docker/docker/client.pyi

```diff
@@ -12,11 +13,25 @@ from docker.models.services import ServiceCollection
 from docker.models.swarm import Swarm
 from docker.models.volumes import VolumeCollection
 
+@type_check_only
+class _Environ(Protocol):
```

Comment:
This is copied from:

https://github.com/python/typeshed/blob/a375953f636561833ae18b00c5c8853817107615/stdlib/cgi.pyi#L35-L37